opentracing-apache-httpclient/src/main/java/io/opentracing/contrib/apache/http/client/TracingHttpClientBuilder.java

```diff
@@ -0,0 +1,94 @@
+package io.opentracing.contrib.apache.http.client;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.apache.http.client.ApacheClientSpanDecorator.StandardTags;
+import io.opentracing.util.GlobalTracer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.apache.http.client.RedirectStrategy;
+import org.apache.http.impl.client.DefaultRedirectStrategy;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.execchain.ClientExecChain;
+
+/**
+ * @author Pavol Loffay
+ */
+public class TracingHttpClientBuilder extends org.apache.http.impl.client.HttpClientBuilder {
```

Comment:
You import this class already, why are you using the fully qualified class name?